src/components/Editor/EditorMenu.js

```diff
@@ -3,29 +3,28 @@ const { isEnabled } = require("devtools-config");
 const { isOriginalId } = require("devtools-source-map");
 const { copyToTheClipboard } = require("../../utils/clipboard");
 
-async function EditorMenu(
+function getMenuItems(
```

Comment:
rejiggered some things so that the editor menu can show different items when blackboxed